src/Payload.php

```diff
@@ -197,50 +197,50 @@ public function __toString()
     /**
      * Determine if an item exists at an offset.
      *
-     * @param mixed $key
+     * @param mixed $offset
      *
      * @return bool
      */
-    public function offsetExists($key)
+    public function offsetExists($offset): bool
```

Comment:
Is there a need to rename arguments? Since PHP 8 there are named arguments, so changing their name might be a breaking change

Reply:
I don't see it as a great breaking chance, we're following the interface as suggested to scope match with it.